CONTRIBUTING.md

```diff
@@ -82,6 +82,37 @@ To fix all linting issues, run:
 $ pnpm lint:fix
 ```
 
+## Releases
+
+To release a new version follow these steps make sure that some prerequisites are met:
+
+- The smoke tests are passing.
+- There are changesets under `.changeset`. Without changesets there should be nothing to release.
```

Comment:
```suggestion
- There are changesets under the `.changeset` directory. Without changesets, there should be nothing to release.
```

Reply:
I actually noticed that but ignored it anyways 🤫. Thanks for being the style guide police! 🚨